assets/styles/app.scss

```diff
@@ -156,6 +176,34 @@ svg {
     align-items: start;
 }
 
+.div-btn {
+    width:60%; 
+    display:flex; 
+    align-items: center;
+    justify-content: center;
+    padding-left: 4.7rem;
```

Comment:
`width: 60%` and `padding-left: 4.7rem`: that's unstable code. avoid fixed width (content should drive the size of the container) and "big" asymetric padding

Reply:
done